### PR TITLE
Latest section component now handles blog posts

### PR DESCRIPTION
**Pull request by monkemedia**

### CODE REVIEW

File: components/browse/LatestSection.vue [hunk 1]
```diff
@@ -58,16 +58,22 @@
...
         }
+
+        return false;
       },
       showMoreLink() {
         return `${this.$tc('showMore')} ${this.contentType.name.toLowerCase()} (${this.total})`;
+      },
+      buttonPath() {
+        return { name: (this.category === 'Blog posts') ? 'blog' : this.category.toLowerCase() };
       }
```
Comment (mirjamverloop) [inline components/browse/LatestSection.vue:77]: Could you get rid of this and use the added "path" to the contentType instead?

File: components/browse/LatestSection.vue [hunk 2]
```diff
@@ -23,7 +23,7 @@
       <b-button
         v-if="total > 4"
         variant="light"
-        :to="localePath({ name: category.toLowerCase() })"
+        :to="localePath(buttonPath)"
```
Comment (mirjamverloop) [inline components/browse/LatestSection.vue:26]: Use `contentType.path` here.

File: lang/en.js [hunk 3]
```diff
@@ -369,6 +369,7 @@ export default {
   "newWindow": "opens in new window",
   "blog": {
     "blog": "Blog",
+    "blogs": "Blogs",
```
[the remark this answers is not in the document]
Comment (monkemedia) [reply lang/en.js:372]: @rwd what would you like me to do here?

[the remark this answers is not in the document]
Comment (rwd) [reply lang/en.js:372]: We should be outputting "Blog posts" but that is not a plural of "Blog", so remove blog.blogs and add blog.posts with English text "Blog posts".

**Pull request merged**